selfservice/strategy/oidc/strategy_registration.go

```diff
@@ -381,9 +395,7 @@ func (s *Strategy) createIdentity(w http.ResponseWriter, r *http.Request, a *reg
 		return nil, nil, s.handleError(w, r, a, provider.Config().ID, i.Traits, err)
 	}
 
-	if orgID := httprouter.ParamsFromContext(r.Context()).ByName("organization"); orgID != "" {
-		i.OrganizationID = uuid.NullUUID{UUID: x.ParseUUID(orgID), Valid: true}
-	}
+	i.OrganizationID = a.OrganizationID
```

Comment:
How are you certain that this is not a regression?

Reply:
Yes, but it did end up changing the code to get the correct OrgID directly from the provider configuration.